### PR TITLE
Tic Tac Toe / Folder Structure

### PR DESCRIPTION
**Pull request by Dorian30**
## Summary
- Took the base code and adapted it to the tree folder structure proposed in the card.
- Defined 3 main components with their respective styles: Board, Game and Square.
- Tested it with npm start.

## Screenshot
image

## Trello Card
https://trello.com/c/PMgm9D0F/32-estructura-de-carpetas

### CODE REVIEW

**Comment by LucasZibell** [comment]
How about moving the `App` folder into a screen folder as `game` or something like this and import the components to use in that screen

File: react-app/src/scss/index.scss [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
-  margin: 0;
-  padding: 0;
-  font-family: sans-serif;
+  font: 14px "Century Gothic", Futura, sans-serif;
+  margin: 20px;
 }
+
+ol, ul {
+  padding-left: 30px;
+}
```
Comment (HarryPerez) [inline react-app/src/scss/index.scss]: Add `EOF` here. (End of file).

File: react-app/src/app/components/Square/styles.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  margin-right: -1px;
+  margin-top: -1px;
+  padding: 0;
+  text-align: center;
+  width: 34px;
+}
+
+.square:focus {
+  outline: none;
+}
```
Comment (HarryPerez) [inline react-app/src/app/components/Square/styles.scss]: Add `EOF` here. (End of file).

File: react-app/src/app/components/Board/style.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.status {
+  margin-bottom: 10px;
+}
+
+.boardRow:after {
+  clear: both;
+  content: "";
+  display: table;
+}
```
Comment (HarryPerez) [inline react-app/src/app/components/Board/style.scss]: Add `EOF` here. (End of file).

File: react-app/src/app/components/Game/index.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        <div className={style.gameInfo}>
+          <div>{/* status */}</div>
+          <ol>{/* TODO */}</ol>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Game;
```
Comment (HarryPerez) [inline react-app/src/app/components/Game/index.js]: Game should be an screen. `Then Board...Squares and all componentes should be inside Game screen.`

Comment (HarryPerez) [reply react-app/src/app/components/Game/index.js]: For example: `/src/app/screens/Game/components/Square/index.js`

**Review by HarryPerez — CHANGES_REQUESTED** [review]

**Comment by HarryPerez** [comment]
Very good!! In this pr we will only check folder structure. In the next PR we will check the whole app

**Review by HarryPerez — APPROVED** [review]

**Pull request merged**